QMLComponents/controls/comboboxbase.cpp

```diff
@@ -82,6 +82,8 @@ void ComboBoxBase::bindTo(const Json::Value& value)
 			index = int(std::distance(values.begin(), itr));
 		}
 	}
+	else if (!selectedValue.empty())
+		_lostValue = selectedValue;
```

Comment:
Bedoel je "lost" value of "last" value?

Reply:
I called it lostValue because of the issue it solves: the dropdown is initialized with a value, but has no possible values (its model is empty). This is due to the fact that the VariablesList it depends on, gets initialized afterwards. In this case (when the dropdown has for values the levels of a VariablesList control), the dropdown has no way of guessing on which control it depends (`values : varList.levels`: the values are set with a list of strings, the levels, but it has no reference of the varList control self). This means that during the initialization of the form, VariablesList might be initialized after the dropdown.
Usually, if the dropdown is set with a non-existing value, it neglects this value (probably it comes from an old version of the analysis, and sets a warning), but in this case we want to keep this value until the form is completely initialized: if in the meantime, the dropdown gets a signal that its model is updated, it should try again to set this `lostValue`.
Maybe `lostValue` is not a good choice, but `lastValue` is definitly not he right name. This value should be used only during the initialization of the form. Some comments in the code would be nice also....